src/test/java/com/smartbear/postman/collection/PostmanCollectionFactoryTest.java

```diff
@@ -13,47 +14,43 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.CoreMatchers.instanceOf;
-import static org.junit.Assert.assertNull;
 
 public class PostmanCollectionFactoryTest {
     private final String COLLECTION_V1_PATH = "/Postman_Echo.postman_collection";
     private final String COLLECTION_V2_PATH = "/Postman_Echo.postman_collection_v2.0";
     private final String COLLECTION_V2_1PATH = "/Postman_Echo.postman_collection_v2.1";
     private final String COLLECTION_FROM_URL_PATH = "/Postman_Echo_from_url.postman_collection_v2.1";
 
-    @Test
-    public void postmanCollectionV1ImportIsNull() throws IOException {
-        PostmanCollection postmanCollection = getCollectionFromFile(
-                getClass().getResource(COLLECTION_V1_PATH).getPath()
-        );
-        assertNull(postmanCollection);
+    @Test(expected = PostmanCollectionUnsupportedVersionException.class)
+    public void postmanCollectionV1ImportIsNull() throws IOException, PostmanCollectionUnsupportedVersionException {
+        getCollectionFromFile(getClass().getResource(COLLECTION_V1_PATH).getPath());
     }
 
     @Test
-    public void postmanCollectionV2ImportsAsV2() throws IOException {
+    public void postmanCollectionV2ImportsAsV2() throws IOException, PostmanCollectionUnsupportedVersionException {
```

Comment:
I think you can declare just `throws Exception` in tests, it is more compact. In tests we don't really care about the exception type, it is only for compiler. For us exception will fail the test and we will see it in test report anyway.